static/templates/actions_popover_content.handlebars

```diff
@@ -14,7 +14,7 @@
     </li>
 
     <li>
-        <a href="#" class="popover_toggle_collapse" data-message-id="{{message.id}}">
+        <a href="#" class="popover_toggle_collapse" data-message-id="{{message.id}}" title="{{t 'Collapse' }} (-)">
```

Comment:
I think for menus we should just put it in the popover itself, instead of in a `title` attribute. (E.g., how chrome and I'm sure other browsers/desktop programs display keyboard shortcuts corresponding to menu items).

That way there's also not an inconsistency with some menu items having them and some not.